doc/how_to_handle_errors.md

```diff
@@ -0,0 +1,29 @@
+# How to proceed with Sentry Errors
+
+We aim to have an [empty list of Errors][1] in Sentry at all times, so with each occurrence in Sentry, we should take action to prevent it from showing again.
+
+The main idea is to have only actionable errors in Sentry because otherwise, they are just noise that could hide important information; [you can read more about errors and how to classify them in GOV.UK developers][2].
+
+## If you are a new developer to the team
+
+1. Subscribe to Sentry errors for Content Data Admin and Content Performance Manager.
```

Comment:
We should mention which team to join in Sentry that has access to these apps. Currently `content-tools-team` for CPM and `govuk-data-informed-team` for the Content Data Admin.

Reply:
Addressed in bace695